encryptio.py

```diff
@@ -1,9 +1,55 @@
 #! /usr/bin/env python3
 
+import argparse
+from enum import Enum
+import os
+from pathlib import Path
+
 authorSignature = 'encryptio.py - Published by Moritz Nentwig and Laura Tzigiannis\n'
 authorSignature += '---------------------------------------------------------------'
 
 ## print header first 
 print("")
 print(authorSignature)
+print("")
+
+## possible encryption algorithms
+class EncryptionAlgorithms(Enum): 
+    aes='AES'
+    des='DES'
+    trippleDes='3DES'
+    idea='IDEA'
+
+    def __str__(self):
+        return self.value
+
+## argument checker
+def check_key(key):
+
+    if not (key.isalpha()):
```

Comment:
In dem Schlüssel können doch auch Zahlen vorkommen oder?

Reply:
I'd like to stick with alphanumeric characters for now.